### PR TITLE
Feature/2493 change logo support

### PR DESCRIPTION
**Pull request by jaapmarcus**
Upload files to /usr/local/hestia/web/images/custom

Clear cache might be required:

Screenshot 2023-05-11 at 23 35 35

### CODE REVIEW

File: web/templates/pages/edit_whitelabel.php [hunk 1]
```diff
@@ -113,6 +113,35 @@ class="form-control"
...
+						<ul>
+							<li>logo.svg <small>(100px x 120px)</small></li>
+							<li>logo.png <small>(100px x 120px)</small></li>
+							<li>logo-header.png <small>(54x x 29px)</small></li>
+							<li>favicon.png <small>(64px x 64px)</small></li>
+							<li>favicon.ico<<small>(16px x 16px)</small></li>
+						</ul>
+					</div>
+					<div class="u-mb10">
+						<input type="checkbox" id="v_update_logo" name="v_update_logo"/>
```
Comment (AlecRust) [inline web/templates/pages/edit_whitelabel.php]:
```suggestion
						<input type="checkbox" id="v_update_logo" name="v_update_logo">
```

**Review by AlecRust — APPROVED** [review]

**Comment by dfag86** [comment]
whitelabel works fine? I made the directory /custom/ because it did not exist, I added the appropriate file sizes, named them as in the manual, cleared the cache, refreshed, but I still see old data. Nothing is updating :(

**Comment by jaapmarcus** [comment]
Run

```
v-update-white-label-logo
```

It should work fine

**Comment by dfag86** [comment]
done but no change :(

**Comment by jaapmarcus** [comment]
And cleared caches Hestia is know for way to long cache on things. You can also try

 /usr/local/hestia/web/images/logo.svg?randomstringhere

And it should clear the cache

We just copy over:

https://github.com/hestiacp/hestiacp/blob/7702ea522c42fcf1da492babb25bbd5b99aa8c50/bin/v-update-white-label-logo#L30-L34

The files to the 1 folder higher

**Comment by dfag86** [comment]
Everything works fine, thanks.
I had the png files copied, and the SVG not, hence the error. Sorry, thread closed :)

**Pull request merged**